forms-flow-web/src/services/PdfService.js

```diff
@@ -22,6 +22,13 @@ export const exportToPdf = ({formId, formName, pdfName}) => {
 };
 
 export const printToPDF = ({ pdfName, formName }) => {
+  let maxWidth = '1440px';
+  if (window.location.href.includes('/task/')) {
+    maxWidth = '1920px';
+  }
+  const appContainer = document.querySelectorAll("div.app-container.container")[0];
+  appContainer.style.maxWidth = maxWidth;
+  appContainer.style.minWidth = maxWidth;
```

Comment:
I guess this is a duplicate line

Reply:
@iman-jamali-fw 
it sets both properties **min-width** and **max-width**. that's the only way to overwrite exiting width setting of container here.